config/webpack.config.js

```diff
@@ -56,52 +59,15 @@ let cfg = {
   module: {
     rules: [
       {
-        test: /\.(png|jpg|ico|svg|gif)$/,
-        use: [
-          {
-            loader: 'file-loader',
-            options: {
-              name: assetName(),
-              esModule: false,
-            },
-          },
-        ],
+        test: /.s?css$/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader', 'sass-loader'],
```

Comment:
This is odd, when using the minicssextractplugin, there is a font error, for what appear to be non-existant fonts:

```js
ERROR in ./src/layouts/govuk.screen.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/css-loader/dist/cjs.js):
Error: Can't resolve '/assets/fonts/light-94a07e06a1-v2.woff2' in '/home/oatman/projects/gds/paas/paas-admin/src/layouts'
```

Reply:
not only can I not find `light-94a07e06a1-v2.woff2` but there's no mention of the word "fonts" or "woff2" in the entire codebase (outside a single jest config option)

Reply:
@paroxp pointed out these are Design System fonts, so the new loader isn't pulling in that properly - looking into it